src/main/java/nextstep/subway/section/domain/Sections.java

```diff
@@ -0,0 +1,75 @@
+package nextstep.subway.section.domain;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
+import lombok.Getter;
+import nextstep.subway.common.exception.ExistException;
+import nextstep.subway.common.exception.NothingException;
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.station.domain.Station;
+
+@Getter
+@Embeddable
+public class Sections {
+
+	@OneToMany(mappedBy = "line", cascade = CascadeType.ALL) //, orphanRemoval = true
+	private final List<Section> sections = new ArrayList<>();
+
+	public List<Station> getStations() {
+		Set<Station> result = new LinkedHashSet<>();
+		for (Section section : this.sections) {
+			result.addAll(section.getStations());
+		}
+		return new ArrayList<>(result);
+	}
+
+	public void createSection(Line line, Station upStation, Station downStation, int distance) {
+		this.sections.add(new Section(line, upStation, downStation, distance));
+	}
+
+	public void add(Section target) {
+		validateSection(target);
+
+		if (isContainStation(target.getUpStation())) {
+			this.sections.stream()
+				.filter(section -> section.getUpStation().equals(target.getUpStation()))
```

Comment:
```suggestion
				.filter(section -> section.equalsUpStation(target.getUpStation()))
```
getter로 값을 꺼내서 비교하지 말고 section 객체에게 물어보는게 어떨까요?

Reply:
네 반영하겠습니다.!!